apps/server/src/schema.graphql

```diff
@@ -104,6 +102,7 @@ type Review {
   reviewerId: String
   isDeclined: Boolean
   submitted: Boolean
+  isDrafted: Boolean
```

Comment:
it's throwing an error when I try to Save Draft. I think this needs to be on ReviewInput, not ReportInput